release/release.go

```diff
@@ -67,6 +67,16 @@ func (o *OS) ForceDevMode() bool {
 	return false
 }
 
+// SupportsClassicConfinement returns true if the current distribution can run snaps using
+// classic confinement.
+func (o *OS) SupportsClassicConfinement() bool {
+	switch o.ID {
```

Comment:
Sorry for not realizing this before.

Can you please rewrite this test so that it looks at `dirs.SnapMountDir` and compares that to "/snap", this way we don't have to maintain anything.